index.html

```diff
@@ -261,9 +261,15 @@ <h1 class="features-title">How can you join?</h1>
             you don’t know much yet, but it won’t be okay to notput in any
             efforts for yourself. We want to value everyone’s time and efforts.
           </p>
-          <a class="btn-join" href="https://realdevsquad.com/join">
-            Join the Squad
-          </a>
+
+          <button class="btn-join" id="button-disable">
+            <a href="https://realdevsquad.com/join"> Join the Squad </a>
+          </button>
```

Comment:
Can you please explain why the anchor was changed to a button?

Reply:
removed button and using only anchor tag